packages/pyolite-kernel/src/worker.ts

```diff
@@ -123,6 +123,21 @@ async function execute(content: any) {
     });
   };
 
+  const publishExecutionError = (ename: any, evalue: any, traceback: any): void => {
+    const bundle = {
+      ename: ename,
```

Comment:
`ename: formatResult(ename),`

etc

Reply:
But this does not seem to work for some reason (leading to `Uncaught TypeError: obj.forEach is not a function`, because `ename.toJs()` does not work as intended)